src/components/menu-list/menu-list.stories.mdx

```diff
@@ -18,7 +18,7 @@ import { MenuList, MenuListItem } from "carbon-react/lib/components/menu-list";
 
 ### Default
 <Preview>
-  <Story name="default">
+  <Story name="default" parameters={{ chromatic: { disable: true }}}>
```

Comment:
we can move `parameters={{ chromatic: { disable: true }}}` into the `<Meta>` to avoid duplication

Reply:
Okay will amend